project/AppDependencies.scala

```diff
@@ -43,7 +43,7 @@ object MainDependencies {
     "uk.gov.hmrc"         %% "play-reactivemongo"             % reactiveMongoVersion,
     "commons-validator"   %  "commons-validator"              % commonsValidatorVersion,
     "uk.gov.hmrc"         %% "govuk-template" % "5.26.0-play-25",
```

Comment:
i don't think it flagged up on the catalogue but this is 10 versions out of date https://github.com/hmrc/govuk-template/releases

Reply:
as you are changing any ways can you change these two numbers to be vals instead like all the other dependencies above rather than the number inline